src/AppV2.js

```diff
@@ -5,19 +5,23 @@ import {hot} from 'react-hot-loader/root';
 import {Route, Switch} from 'react-router-dom';
 import Header from 'v2/components/Header';
 import NavBar from 'v2/components/NavBar';
+import Footer from 'v2/components/Footer';
 import theme from 'v2/theme';
+import socket from 'v2/stores/socket';
+
+socket.init();
```

Comment:
this seems like it needs to turn into an event-based rather than a static call based on @mvines PR for endpoint changes